frontend/src/pages/ProjectSettings/ProjectSettings.tsx

```diff
@@ -91,128 +94,133 @@ const ProjectSettings = () => {
 				<title>Project Settings</title>
 			</Helmet>
 
-			<Box>
-				<Box style={{ maxWidth: 560 }} my="40" mx="auto">
-					<Heading mt="16" level="h4">
-						Project Settings
-					</Heading>
-					<div className={styles.tabsContainer}>
-						<ProjectSettingsContextProvider
-							value={{
-								allProjectSettings,
-								setAllProjectSettings,
-								loading,
+			<Box style={{ maxWidth: 560 }} my="40" mx="auto">
+				<Heading mt="16" level="h4">
+					Project Settings
+				</Heading>
+				<div className={styles.tabsContainer}>
+					<ProjectSettingsContextProvider
+						value={{
+							allProjectSettings,
+							setAllProjectSettings,
+							loading,
+						}}
+					>
+						<Tabs
+							activeKeyOverride={params.tab ?? 'sessions'}
+							onChange={(key) => {
+								navigate(`/${project_id}/settings/${key}`)
 							}}
-						>
-							<Tabs
-								activeKeyOverride={params.tab ?? 'sessions'}
-								onChange={(key) => {
-									navigate(`/${project_id}/settings/${key}`)
-								}}
-								noHeaderPadding
-								noPadding
-								id="settingsTabs"
-								tabs={[
-									{
-										key: 'sessions',
-										title: 'Session replay',
-										panelContent: (
-											<Stack>
-												<Box
-													display="flex"
-													gap="8"
-													justifyContent="space-between"
-													alignItems="center"
+							noHeaderPadding
+							noPadding
+							id="settingsTabs"
+							tabs={[
+								{
+									key: 'general',
```

Comment:
Was there a reason for moving this to the first tab?

Reply:
Matching the mocks here: https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?node-id=10517%3A526107&mode=dev

Didn't add the icons though